src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -951,15 +943,170 @@ The description of the configuration options for the remaining ProActive compone
 [[_shell_Starting_ProActive_using_Shell_scripts]]
 ==== Starting ProActive Using Shell Scripts
 
-===== ProActive server installation
 
-After configuring the appropriate settings as outlined in the previous section, execute the `proactiveInstaller.sh` script along with the `PASSPHRASE` used to encrypt the passwords.
+After configuring the necessary settings as described in the previous section, you are ready to begin the installation of the ProActive server using the `proactiveInstaller.sh` script.
+
+The `proactiveInstaller.sh` script simplifies the installation and configuration process by automating key tasks such as password encryption and offering flexibility through offline and upgrade modes.
+
+To get an overview of all available options and some examples of installation commands, you can access the help menu by running:
+
+----
+$ sudo ./proactiveInstaller.sh -h
+----
+
+This script supports three different modes for password encryption: via command-line arguments, interactive, and automatic (using a password file).
+It ensures that you can tailor the installation method based on your environment and security requirements.
+
+The help menu provides a detailed description of the available options:
+
+- **CLI Arguments Mode** (`-p passphrase -admin_pwd admin_password -user_pwd user_password`): Allows you to pass the passphrase and passwords directly as command-line arguments for minimal installation setups.
+- **Interactive Mode** (`-i`): Prompts you to enter passwords and passphrases interactively, followed by automatic installation.
+- **Automatic Mode** (`-a -f file_path`): Uses a predefined file with clear-text passwords for installation, ideal for automated deployments.
+
+Additional options include:
+
+- **Offline Mode** (`-o`): Ensures the installation process can be completed without internet access.
+- **Upgrade Mode** (`-u`): Allows upgrading an existing ProActive installation.
+
+Below is the help output that showcases the available options and example commands:
+
+----
+$ sudo ./proactiveInstaller.sh -h
+ This script automates the installation and configuration process for ProActive, including password encryption and optional offline or upgrade modes.
+ The script supports 3 modes for passwords encryption: interactive, automatic, and CLI arguments.
+
+ Usage: ./proactiveInstaller.sh [-i | -a -f file_path | -h | -p passphrase [-admin_pwd admin_password -user_pwd user_password] | -u] [-o]
+ (Only one of the following installation methods can be used at a time: -i, -a, -p, or -h):
```

Comment:
consider using backticks to highlight command options

Reply:
not applicable, this is bash code so they will be shown in bash code formatting block.